### PR TITLE
VideoPlayer: stop clock while display is lost

### PR DESCRIPTION
**Pull request by FernetMenta**
see title

@popcornmix ping

### CODE REVIEW

**Comment by popcornmix** [comment]
This works for first video I play. Second video just hangs on start and I get these errors:
  ERROR: CDVDAudio::AddPacketsRenderer - timeout adding data to renderer

http://paste.ubuntu.com/17271029/

**Comment by FernetMenta** [comment]
your log doesn't seem to be related to this change here. where do you see a relationship?

**Comment by popcornmix** [comment]
I don't have an explanation. I have double checked - this PR stops me being able to play a second file.

In fact with "delay after refresh rate change" disabled, it hangs even on first play of the file:
http://paste.ubuntu.com/17273264/

File: xbmc/cores/VideoPlayer/DVDClock.cpp [hunk 1]
```diff
@@ -100,14 +101,41 @@ double CDVDClock::GetVsyncAdjust()
...
+    else
+      m_speedAfterPause = DVD_PLAYSPEED_PAUSE;
+
+    SetSpeed(DVD_PLAYSPEED_PAUSE);
+    m_paused = true;
+  }
+  else if (!pause && m_paused)
+  {
+    SetSpeed(m_speedAfterPause);
+    m_paused = false;
```
Comment (FernetMenta) [inline xbmc/cores/VideoPlayer/DVDClock.cpp]: @popcornmix those 2 statements are in wrong order. SetSpeed won't start the clock in m_paused is true. I will correct it

**Comment by FernetMenta** [comment]
jenkins build this please

**Comment by popcornmix** [comment]
Updated PR looks good now.

**Pull request merged**